BitmarkRegistry/AppDelegate.swift

```diff
@@ -107,6 +107,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     registerAPNSSubject.onCompleted()
   }
 
+  func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
+    guard let _ = url.scheme, let host = url.host else { return false }
+
+    switch host {
+    case "authorization":
+      let verificationLink = String(url.path.dropFirst())
+      Global.verificationLink = verificationLink
+
+      if Global.currentAccount == nil {
+        showAuthorizationRequiredAlert()
```

Comment:
after user passes the authorization alert, it won't continue the flow of opening the url

Reply:
I marked it with Global.verificationLink; 
so after user login successfully and go to properties VC; if Global.verificationLink is existing, it will lead to open the url.

Reply:
`showAuthorizationRequiredAlert(completion:)` sounds more elegant?

Reply:
I have to assign Global.verificationLink in both cases; it's likely sentence: I get the verification link from opening URL.

and showAuthorizationRequiredAlert don't really do thing, it just show the alert to user.

Reply:
ok